package.json

```diff
@@ -41,11 +41,11 @@
     "eslint-loader": "1.6.1",
     "eslint-plugin-react": "6.8.0",
     "html-webpack-plugin": "2.24.1",
-    "jade": "1.11.0",
-    "jade-loader": "0.8.0",
     "json-loader": "0.5.4",
     "node-sass": "4.1.1",
     "postcss-loader": "1.2.1",
+    "pug": "2.0.0-beta6",
+    "pug-loader": "2.3.0",
     "react-onclickoutside": "4.5.0",
```

Comment:
I've tried it on https://github.com/Automattic/simplenote-electron/pull/464.
Using as an HOC for the NavigationBar, while keeping it as a mixin for everything else, and I haven't noticed any error or incorrect behaviour.

(Also, just for information: that module is now at version 5.8.3, but since 5.0 it doesn't support mixins anymore.)